src/sentry/static/sentry/app/views/organizationDashboard/utils/getEventsUrlFromDiscoverQueryWithConditions.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * Generate a URL to the events page for a discover query that
+ * contains a condition.
+ *
+ * @param {Object} query The discover query object
+ * @param {String[]} values A list of strings that represent field values
+ *   e.g. if the query has multiple fields (browser, device), values could be ["Chrome", "iPhone"]
+ * @return {String} Returns a url to the "events" page with any discover conditions tranformed to search query syntax
+ */
+import {zipWith} from 'lodash';
+
+import {OPERATOR} from 'app/views/organizationDiscover/data';
+import {getEventsUrlPathFromDiscoverQuery} from 'app/views/organizationDashboard/utils/getEventsUrlPathFromDiscoverQuery';
+
+export function getEventsUrlFromDiscoverQueryWithConditions({
+  values,
+  query,
+  selection,
+  organization,
+}) {
+  return getEventsUrlPathFromDiscoverQuery({
+    organization,
+    selection,
+    query: {
+      ...query,
+      conditions: [
+        ...query.conditions,
+        // For each `field`, create a condition that joins it with each `rowObject.name` value (separated by commas)
+        // e.g. fields: ['browser', 'device'],  rowObject.name: "Chrome, iPhone"
+        //      ----> [['browser', '=', 'Chrome'], ['device', '=', 'iPhone']]
+        ...zipWith(query.fields, values, (field, value) => [
+          field,
+          OPERATOR.EQUAL,
+          value === null ? '""' : `"${value}"`,
```

Comment:
Could `value` contain `"` that need to be escaped?

Reply:
Potentially, but I don't think our search backend supports escaping anyway, can you confirm @macqueen?

Reply:
we actually do support escaping quotes:
https://github.com/getsentry/sentry/blob/master/tests/sentry/api/test_event_search.py#L272-L278